### ISSUE TITLE
Crosshair close price readout width

### ISSUE THREAD

**Issue by sneurlax** [andredumas/techan.js#187]
Hello!  Techan.js is great, I'm loving it, really nice.  Thank you for this!

First off, this is not an issue with techan itself, but I don't know how better to communicate with the creator and/or its users.  I apologize if it's a major no-no to open an issue in order to ask a question!

I have adapted the bigChart shown on the techanjs.org front page for http://bpoch.info/ (DNS isn't updated yet, preview at http://bpoch-info.nfshost.com), whose code can be found in [sneurlax/bpoch](https://github.com/sneurlax/bpoch/tree/master/www/info)

Basically: on currency pairs other than USD_BTC the close price reads out to a full 8 decimals.  I have not found where I can specify the readout's width in order to accommodate all of those decimal points.

If I need to compile my work into a JSFiddle (or likewise) to make troubleshooting easier I can of course do that legwork.  I appreciate your time!

**Comment by sneurlax** [comment]
I sorted the date problem myself and no longer really need to flip the axes :)

**Comment by sneurlax** [comment]
I'm closing this issue, sorry for opening it, I figured out most of my problems!  I still have some but I'll take them to StackOverflow or somewhere similar
